packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -0,0 +1,299 @@
+              () => wsHandler.unsubscribe(payload.input),
+              (message) => {
+                /**
+                 * If the error in on the subscription, next requests will try to subscribe
+                 * If the error happens during a subscription, and is only eventual, can be ignored
+                 * If the error happens during a subscription, and the subscription stop receiving messages, the unresponsiveTimeout will take care of it (unsubs/subs)
+                 */
+                if (wsHandler.isError(message)) {
+                  logger.error(`WS: Subscription Error: ${JSON.stringify(message)}`)
+                  return false
+                }
+                return getSubsId(wsHandler.subsFromMessage(message)) === subscriptionKey
+              },
+            )
+            .pipe(
+              withLatestFrom(state$),
+              map(([message, state]) => {
+                // TODO: Use merge map to not discard the first message
+                if (!state.ws.subscriptions[subscriptionKey]?.active) {
+                  return subscribed(payload)
+                }
+                const input = state.ws.subscriptions[subscriptionKey]?.input || {}
+                if (!input) logger.warn(`WS: Could not find subscription from incoming message`)
+                return messageReceived({ message, subscriptionKey, input })
+              }),
+              takeUntil(
+                // unsubscribe or disconnected
+                merge(
+                  action$.pipe(
+                    filter(unsubscribe.match),
+                    filter((a) => getSubsId(a.payload.subscriptionMsg) === subscriptionKey),
+                  ),
+                  action$.pipe(
+                    filter(disconnected.match),
+                    filter((a) => a.payload.config.connectionInfo.key === connectionKey),
+                  ),
+                ),
+              ),
+              endWith(unsubscribed(payload)),
+            )
+          )
+      )
+
+      const messages$ = action$.pipe(
+        filter(messageReceived.match),
+        filter((action) => wsHandler.filter(action.payload.message)),
+        mergeMap(async (action) => {
+          const response = wsHandler.parse(action.payload.message)
+          if (!response) return action
+          const adapterResponse = wsHandler.toAdapterResponse || ((result: any) => Requester.success('1', { data: { result } }))
+          const execute: Execute = () => Promise.resolve(adapterResponse(response))
+          const cache = await withCache(execute)
+          const input = {
+            ...action.payload.input,
+            data: {
+              ...action.payload.input.data,
+              maxAge: -1
+            },
+            debug: {
+              ws: true
+            }
+          }
+          await cache(input)
+          return action
+        }),
+        catchError((error) => of(error).pipe(log(`WS: ${error.message}`))),
```

Comment:
If we have an error here, this will close the stream as you just log and propagate/throw the error again.

Remove this line, and add a try/catch in the `mergeMap` above, where we'll catch the cache errors, and log them for now.